src/replicache.test.ts

```diff
@@ -2165,7 +2165,7 @@ test('experiment KV Store', async () => {
   store.resetCounters();
 
   await rep.persist();
-  expect(store.readCount).to.equal(1, 'readCount');
+  expect(store.readCount).to.equal(2, 'readCount');
```

Comment:
One more read because we read to check for the clients...

Maybe we can bake this into the write transaction. Let me look into that.

Reply:
> Maybe we can bake this into the write transaction. Let me look into that.

No. Then we have to bake that into `updateClients` which seems too strange,

Reply:
This tests is a bit annoying.

Reply:
Agreed. Maybe change the test to check what is in the experimental kv store instead of counting the reads/writes/close?